configs/kueue/dev/servicemonitor.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app.kubernetes.io/instance: kueue
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: kueue
+    argocd.argoproj.io/instance: kueue
+    control-plane: controller-manager
+  name: kueue-controller-manager-metrics-service
+  namespace: kueue-system
```

Comment:
Does this need to be configurable somewhere, or is the assumption that Kueue will always operate in the `kueue-system` Namespace?

Reply:
Conventionally, that is the recommended practice when included in a helm chart. This is ultimately a bit of a bandaid until Kueue resolve the ServiceMonitor bug within their Helm chart. We can remove here though and apply it via the Argo deployment.